snmp/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG - snmp
 
+## 5.11.0 / 2022-12-09
+
+* [Added] Add topology metadata e2e test (aos_lldp). See [#13373](https://github.com/DataDog/integrations-core/pull/13373).
+* [Fixed] Update dependencies. See [#13478](https://github.com/DataDog/integrations-core/pull/13478).
+* [Fixed] Remove invalid 'service' field in snmp example config. See [#13341](https://github.com/DataDog/integrations-core/pull/13341).
+* [Fixed] Rename check instance argument to _. See [#13384](https://github.com/DataDog/integrations-core/pull/13384).
```

Comment:
```suggestion
```

This should actually have had the `no-changelog` label, as it doesn't change behavior.